packages/legend-vscode-extension-dependencies/src/index.ts

```diff
@@ -91,8 +160,10 @@ export {
   guaranteeNonNullable,
   guaranteeType,
   isBoolean,
+  NetworkClientError,
```

Comment:
are we good with removing the graph manager logic in this module?

Reply:
Yeah, I just removed the graph logic from it since that now lives in the vscode client code.